src/api/boards/getBoards.ts

```diff
@@ -0,0 +1,41 @@
+import { IHttp, IModify, IPersistence, IRead } from "@rocket.chat/apps-engine/definition/accessors";
+import { IRoom } from "@rocket.chat/apps-engine/definition/rooms";
+import { UIKitViewSubmitInteractionContext, ButtonStyle } from "@rocket.chat/apps-engine/definition/uikit";
+import { IUIKitViewSubmitIncomingInteraction } from "@rocket.chat/apps-engine/definition/uikit/UIKitIncomingInteractionTypes";
+import { IUser } from "@rocket.chat/apps-engine/definition/users";
+import { getAccessTokenForUser } from "../../storage/users";
+import { ModalsEnum } from "../../enums/Modals";
+import { HttpStatusCode } from "@rocket.chat/apps-engine/definition/accessors";
+import { Block } from "@rocket.chat/ui-kit";
+import { getMiroBoardsUrl } from "../../lib/const";
+import { Texts } from "../../enums/Texts";
+
+export async function getBoards({ context, data, room, read, persistence, modify, http }: { context: UIKitViewSubmitInteractionContext; data: IUIKitViewSubmitIncomingInteraction; room: IRoom; read: IRead; persistence: IPersistence; modify: IModify; http: IHttp }) {
```

Comment:
```suggestion
interface IGetBoardsParams {
    context: UIKitViewSubmitInteractionContext; 
    data: IUIKitViewSubmitIncomingInteraction; 
    room: IRoom; 
    read: IRead; 
    persistence: IPersistence; 
    modify: IModify; 
    http: IHttp;
}

export async function getBoards({ context, data, room, read, persistence, modify, http }: IGetBoardsParams) {
```
It's easier to read and maintaining.

Reply:
Sure sir, I will now move interfaces to a separate file and use them.